libraries/math/package.json

```
@@ -34,11 +34,11 @@
     "api-report"
   ],
   "peerDependencies": {
-    "@minecraft/server": "^1.15.0"
+    "@minecraft/server": "^2.0.0"
```

Thanks for making this change!

Minor feedback to address, we will want to support both the 1 and 2 major version ranges, since the primary Vector2 and Vector3 dependency hasn't changed, so could you change the specification here and below to:

```suggestion
    "@minecraft/server": " ^1.15.0 || ^2.0.0"
```

---

Done 👍